cmd/dkg.go

```diff
@@ -0,0 +1,52 @@
+// Copyright © 2022 Obol Labs Inc.
+//
+// This program is free software: you can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by the Free
+// Software Foundation, either version 3 of the License, or (at your option)
+// any later version.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of  MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+// more details.
+//
+// You should have received a copy of the GNU General Public License along with
+// this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package cmd
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"github.com/obolnetwork/charon/dkg"
+)
+
+func newDKGCmd(runFunc func(context.Context, dkg.Config) error) *cobra.Command {
+	var config dkg.Config
+
+	cmd := &cobra.Command{
+		Use:   "dkg",
+		Short: "Create distributed validator key shares by participating in a DKG ceremony",
+		Long: `Participate in a distributed key generation ceremony for a specific cluster definition that creates
+distributed validator key shares and a final cluster lock configuration. Note that all other cluster operators should run
+this command at the same time.`,
```

Comment:
```suggestion
this command at the same time in order to participate in the ceremony.`,
```

Reply:
this feels unnecessary

Reply:
i reason i say so is because when i read `... at the same time` for the first time, it looked to me as if everyone should be perfectly synced to the clock to join a ceremony which i suppose is not the case.

Reply:
sure, but your suggestion doesn't clarify that.

Reply:
And they do need to run it at the same time, they just do not need to start it as the same time 😉

Reply:
yeah i mean this: they can start at different times